lib/blog-feed-loader.js

```diff
@@ -1,2 +1,6 @@
-// TODO: Replace this with a real implementation.
-module.exports = require('../test/browser/stub-blog-feed-loader');
+if (process.env.NODE_ENV !== 'production' &&
+    require('./config').IN_TEST_SUITE) {
```

Comment:
This ensures that tests don't error if the developer is offline and will also make it easier for @mmmavis to implement #953, as the homepage component will always load the fake blog posts in tests.